docs/releases.md

```diff
@@ -20,6 +20,10 @@ Check configuration of the resolvers plugin to see where they are located.
 2. Remove @graphback/codegen-resolvers from your package and the config.
 3. Use Graphback runtime Getting Started Guide 
 4. Change your client output format from `gqlwithfragment` to `graphql`
+5. Change your client output path to be the output path of generated file
+6. Change your documents path in graphback config to match new `outputPath`
+7. Delete previously generated client documents and rerun generate command
+8. Be sure to correct your documents path in your codegen config if you use `graphql-codegen`
```

Comment:
GraphQL-Code-Generator - codegen is internal slang we use as shortcut

Reply:
All right